### PR TITLE
Added Signal Shift description to Ejection/Discharge/Charm

### PR DESCRIPTION
**Pull request by pianoblook**

### CODE REVIEW

File: Reference.md [hunk 1]
```diff
@@ -3011,8 +3011,11 @@ There are a lot of different moves that involve cluing trash. [Here is a handy f
 
 ## Level 17 - Rarely Used Strategies - Ejection
 
-- *Ejection* refers to a move like a *Bluff*, but instead of playing an unrelated card from *Finesse Position*, someone plays an unrelated card from their *Second Finesse Position*. For example, if a player's hand is completely unclued, their *Second Finesse Position* is slot 2.
-- Just like a *Bluff*, an *Ejection* can only be performed on the very next player.
+- *Ejection* refers to a type of move like a *Bluff*, but instead of playing an unrelated card from *Finesse Position*, someone plays an unrelated card from their *Second Finesse Position*. For example, if a player's hand is completely unclued, their *Second Finesse Position* is slot 2.
+- *Ejections* arise when playing *Finesse Position* would cause *Information Desynchronization* about the initial clue. Thus, playing *Second Finesse Position* serves as a unique way to correctly *‘Shift’* the signal of certain clues.
+- This *Signal Shift* can take a variety of forms: *Play* —-> *Save* (*5 Color Ejection*), *Save* -—> *Play* (*Rank Choice Ejection*), *Save* —-> *Trash* (*Bad Chop Move Ejection*), and *Fix* —-> *Play* (*Double Play Ejection*).
```
Comment (Zamiell) [inline Reference.md:3023]: in this line we are inundated with every signal shift.
do we need to know that information right now?
is it better if we move this information into each section?

Comment (pianoblook) [reply Reference.md:3023]: Yeah I can just add a quick note under each ejection/discharge instead - sounds good

**Review by pianoblook — COMMENTED** [review]
Broke it up a bit

File: Reference.md [hunk 2]
```diff
@@ -3018,23 +3018,27 @@ There are a lot of different moves that involve cluing trash. [Here is a handy f
 
 ## Level 17 - Rarely Used Strategies - Ejection
 
-- *Ejection* refers to a move like a *Bluff*, but instead of playing an unrelated card from *Finesse Position*, someone plays an unrelated card from their *Second Finesse Position*. For example, if a player's hand is completely unclued, their *Second Finesse Position* is slot 2.
-- Just like a *Bluff*, an *Ejection* can only be performed on the very next player.
+- *Ejection* refers to a type of move like a *Bluff*, but instead of playing an unrelated card from *Finesse Position*, someone plays an unrelated card from their *Second Finesse Position*. For example, if a player's hand is completely unclued, their *Second Finesse Position* is slot 2.
+- *Ejections* arise when playing *Finesse Position* would cause *Information Desynchronization* about the initial clue. Thus, playing *Second Finesse Position* serves as a unique way to correctly *‘Shift’* the signal of certain clues.
```
Comment (Zamiell) [inline Reference.md:3022]: no quotes around shift

File: Reference.md [hunk 3]
```diff
@@ -3071,12 +3076,14 @@ There are a lot of different moves that involve cluing trash. [Here is a handy f
 
 ## Level 17 - Rarely Used Strategies - Discharge
 
-- *Discharge* is similar to *Ejection*, except instead of blind-playing the *Second Finesse Position* card, the player plays the *Third Finesse Position* card.
-- Just like a *Bluff*, a *Discharge* can only be performed on the very next player.
+- *Discharge* is similar to *Ejection*, except the player plays their *Third Finesse Position*.
+- *Discharges* arise when playing *First* **or** *Second Finesse Position* would cause *Information Desynchronization* about the initial clue. Thus, playing *Third Finesse Position* serves as a unique way to correctly *‘Shift’* the signal of certain clues.
```
Comment (Zamiell) [inline Reference.md:3080]: same here

File: Reference.md [hunk 4]
```diff
@@ -3108,26 +3116,29 @@ There are a lot of different moves that involve cluing trash. [Here is a handy f
...
   - Red 1 is played on the stacks.
   - It is the *Early Game* and the *Low Score Phase*.
   - Alice clues red to Cathy, touching a new red 2 on slot 2.
   - Cathy hand is, from newest to oldest: `blue 5, red 2, blue 3, blue 2`
   - Bob sees that Alice could have cleanly *5 Pulled* Cathy's red 2. (It is globally known that Alice had another *Play Clue* to give, so a number 5 clue would not have been interpreted as a *5 Stall*.)
   - Bob knows that this signals a blind-play:
-    - A *First Finesse Position* blind-play would look like a *Bluff* on a red 2 or a red 3, which would be a *Lie*.
-    - A *Second Finesse Position* blind-play would look like a *5 Color Ejection* on a red 5, which would be a *Lie*.
-    - A *Third Finesse Position* blind-play would look like an *Unknown Trash Discharge* on a red 1, which would be a *Lie*.
+    - A *First Finesse Position* blind-play would look like a *Bluff* on a red 3, which would be a lie.
```
Comment (Zamiell) [inline Reference.md:3139]: lie should be written as *Lie*

Comment (pianoblook) [reply Reference.md:3139]: yeah didn't mean to change that, weird - one sec

**Comment by Zamiell** [comment]
are we including any graphics or is this only a text-based description?

**Comment by pianoblook** [comment]
I figured it would be weird to put a graphic in the main reference page. But I could add it as a separate page + link?

**Comment by Zamiell** [comment]
is it weird? thats what done on e.g. the clandestine finesse section

**Comment by pianoblook** [comment]
Not sure where a single graphic would make sense to go, since the one I made (a modified version) covers Ejections + Discharges +Charms. Maybe its own little section?

**Comment by Zamiell** [comment]
um i guess

**Review by pianoblook — COMMENTED** [review]
Added Signal Shifting reference (about to upload the image too). A bit bulky upfront, but also reduces necessary text in each subsequent section (now trimmed)

**Review by pianoblook — COMMENTED** [review]
found the white space

**Comment by Zamiell** [comment]
thx

**Pull request merged**